### PR TITLE
Filter only for the main tag the integration tests

### PR DESCRIPTION
**Pull request by bogdandrutu**
Right now when we add the release tags ~40 tags, they will trigger builds for all of them which are duplicated, and we want to trigger the build only for the main tag.

Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

**Comment by bogdandrutu** [comment]
@tigrannajaryan @owais ping on this

File: .circleci/config.yml [hunk 1]
```diff
@@ -245,7 +245,7 @@ workflows:
       - integration-tests:
           filters:
             tags:
-              only: /.*/
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/
```
Comment (tigrannajaryan) [inline .circleci/config.yml:248]: Can you clarify why the change in the PR description? I am not fluent with CircleCI config to understand what this fixes.

Comment (bogdandrutu) [reply .circleci/config.yml:248]: Done.

**Comment by bogdandrutu** [comment]
@tigrannajaryan PTAL

**Review by tigrannajaryan — APPROVED** [review]
